centreon/tests/e2e/features/Poller-configuration/01-poller-configuration/index.ts

```diff
@@ -124,6 +123,7 @@ Then('I am redirected to generate page', () => {
 });
 
 Then('the selected poller names are displayed', () => {
+  cy.reload()
```

Comment:
why do you reload the page ?

Reply:
after the redirection cypress always access to the old iframe , so cy.reload force him to detect and  use the new one

Reply:
then you may need to retry `getIframeBody` until input is available
Because doing a reload does not reflect what a user does on the UI

Reply:
but idk if there is any other alternative i would be happy to implement it